### ISSUE TITLE
Unexpected `interface name must start with a capitalized I`

### ISSUE THREAD

**Issue by danzGentrack** [palantir/tslint#4630]
### Bug Report

- __TSLint version__:5.15.0
- __TypeScript version__:5.15.0
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
interface IS3FileInfo {
    name: string;
}
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended"
    ],
    "jsRules": {},
    "rules": {},
    "rulesDirectory": []
}

```

#### Actual behavior
ERROR: a.ts:1:11 - interface name must start with a capitalized I
#### Expected behavior
The interface name has a capitalized I? tslint-5.14 doesn't report the error.

**Comment by adidahiya** [comment]
sorry about that, it looks like a regression in 5.15.0, duplicate of #4624
